.gitignore

```diff
@@ -14,4 +14,4 @@ target/
 
 # nextflow specific ignores
 .nextflow*
-work
```

Comment:
Undo this change

Reply:
I see that work is included in the .gitignore file. I haven't made any changes to the original .gitignore file of biobox. Could this be an old commit?

Reply:
No, you removed a newline at the end of this file